src/parsers/gas.ts

```diff
@@ -255,24 +267,21 @@ export const parseGasFeeParam = (weiAmount: string): GasFeeParam => {
 /**
  * Transform EIP1559 params into a `GasFeeParams` object
  * @param option - Speed option
- * @param currentBaseFee - Block base fee
  * @param maxFeePerGas - `maxFeePerGas` value in gwei unit
  * @param maxPriorityFeePerGas - `maxPriorityFeePerGas` value in gwei unit
- * @param confirmationTimeByPriorityFee - ConfirmationTimeByPriorityFee object
+ * @param blocksToConfirmation - blocksToConfirmation object
```

Comment:
Nit: did u want the second `blocksToConfirmation` to be capitalized to refer to the `BlocksToConfirmation` entity definition